File: DofusSpa/src/app/_models/resource-recipe.ts
import { Resource } from "./resource";

export interface ResourceRecipe{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    quantity : number,
    level : number,
    type: string,
    resourceID : number,
    resource : Resource
}

File: DofusApi/Models/ResourceRecipe.cs
namespace DofusApi.Models
{
    public class ResourceRecipe
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string ImgUrl { get; set; }
        public int Quantity { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public int ResourceID { get; set; }
    }
}